lib/stories.js

```diff
@@ -73,8 +73,9 @@ const Wrapper = styled.div`
 
 const Nav = styled.div`
   position: sticky;
-  top: 10.5rem;
-  height: calc(100% - 10.5rem);
+  top: 1em;
```

Comment:
I'd suggest `top: 5em` so that the title of the page `Glitch Shared Components` has more prominence than the nav

![image](https://user-images.githubusercontent.com/519336/68407278-492ebd00-0151-11ea-88cd-34ab3f1143ca.png)